tests/testing/LocalBearTestHelperTest.py

```diff
@@ -117,13 +119,26 @@ def create_arguments(filename, file, config_file):
                                   "print('hello stderr', file=sys.stderr)"])
                 return '-c', code
 
+        def new_process_output(output, filename=None, file=None,
+                               **process_output_kwargs):
+            code = '\n'.join(['import sys',
+                              "print('hello stdout')",
+                              "print('hello stderr', file=sys.stderr)"])
+            print(code)
```

Comment:
what are you doing here? 😅

Reply:
@Makman2 Patching the bear with the new_process_output, it will print the code in the command line on test fail

Reply:
but we want a new feature that works in general, not just for some specific test^^ Please modify `execute_bear` like I always told you ;)